docs/structure_of_api_responses/ansari_website_api_structure_of_a_request_sent_using_zrok.json

```diff
@@ -0,0 +1,76 @@
+{
+	"scope": {
+		"type": "http",
+		"asgi": {
+			"version": "3.0",
+			"spec_version": "2.4"
+		},
+		"http_version": "1.1",
+		"server": ["127.0.0.1", 8000], // When running locally
+		"client": ["127.0.0.1", 11563], // The port here changes dynamically
```

Comment:
Is there a better way of handling this?

Reply:
If "this" refers to the "port changes dynamically" comment, then I don't know...

To be honest, I don't even know why this port changes ... even though I've bounded Ansari frontend with port 3000 ... unless "client" here refers to something else, since origin [does indeed have](https://github.com/ansari-project/ansari-backend/blob/2f9eeb223bb6ad74ce0411128a20c47aecd77368/docs/structure_of_api_responses/ansari_website_api_structure_of_a_request_sent_using_zrok.json#L28) port 3000 ...